pyproject.toml

```diff
@@ -48,16 +48,31 @@ select = [
     "W", # pycodestyle Warning
     # PYI: only enable rules that always autofix, avoids duplicate # noqa with flake8-pyi
     # See https://github.com/plinss/flake8-noqa/issues/22
-    "PYI009", # use `...`, not `pass`, in empty class bodies
-    "PYI010", # function bodies must be empty
-    "PYI012", # class bodies must not contain `pass`
-    "PYI013", # non-empty class bodies must not contain `...`
-    "PYI016", # duplicate union member
-    "PYI020", # quoted annotations are always unnecessary in stubs
-    "PYI025", # always alias `collections.abc.Set` as `AbstractSet` when importing it
-    "PYI032", # use `object`, not `Any`, as the second parameter to `__eq__`
-    "PYI055", # multiple `type[T]` usages in a union
-    "PYI058", # use `Iterator` as the return type for `__iter__` methods
+    "PYI009", # Empty body should contain `...`, not pass
+    "PYI010", # Function body must contain only `...`
+    "PYI012", # Class bodies must not contain `pass`
+    "PYI014", # Only simple default values allowed for arguments
+    "PYI015", # Only simple default values allowed for assignments
+    "PYI020", # Quoted annotations should not be included in stubs
+    "PYI032", # Prefer `object` to `Any` for the second parameter to `{method_name}`
+    "PYI058", # Use `{return_type}` as the return value for simple `{method}` methods
+    # PYI rules that sometimes autofix, but we should always be able to manually fix,
+    # so there's still no conflict/duplicate with flake8-pyi
+    "PYI013", # Non-empty class bodies must not contain `...`
+    "PYI016", # Duplicate union member `{}`
+    "PYI025", # Use `from collections.abc import Set as AbstractSet` to avoid confusion with the `set` builtin
+    "PYI030", # Multiple literal members in a union. Use a single literal, e.g. `Literal[{}]`
+    "PYI036", # Star-args in `{method_name}` should be annotated with `object`
+    "PYI044", # `from __future__ import annotations` has no effect in stub files, since type checkers automatically treat stubs as having those semantics
+    "PYI055", # Multiple `type[T]` usages in a union. Combine them into one, e.g., `type[{union_str}]`.
+    "PYI062", # Duplicate literal member `{}`
+    # Rules that have autofixes, but we sometimes disable on a per-line basis
+    # "PYI026", Waiting for this mypy bug to be fixed: https://github.com/python/mypy/issues/16581
+    # "PYI029", __str__ and __str__ sometimes have to be used for classes other than `object`
+    # Sometimes we still wanna be explicit about a default value nonetheless
+    # "PYI011", Also has false-positives with `_typeshed.sentinel`
+    # "PYI053", Also removes `Literal[{string_too_long}]` from an annotation
```

Comment:
We noqa that one twice in nanoid stubs for the paramether `alphabet: str = "_-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"`

Ruff issue for the bad autofix in `Literal`: https://github.com/astral-sh/ruff/issues/12995